extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -20,11 +20,13 @@ function getBuiltinCommands(shell: string): string[] | undefined {
 		if (cachedCommands) {
 			return cachedCommands;
 		}
+		// fixes a bug with file/folder completions brought about by the '.' command
+		const filter = (cmd: string) => cmd && cmd !== '.';
```

Comment:
We do want `.`, I didn't test but assume it's not working based on this. Tracked in https://github.com/microsoft/vscode/issues/234351